src/mono/netcore/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.wasm.xml

```diff
@@ -0,0 +1,7 @@
+<linker>
+  <assembly fullname="System.Private.CoreLib">
+    <type fullname="System.Numerics.Vector">
+      <method signature="System.Boolean get_IsHardwareAccelerated()" body="stub"/>
+    </type>
+  </assembly>
+</linker>
```

Comment:
I don't see this file referenced anywhere.  Is it supposed to be?

Reply:
Ah, I guess it's pulled in by the `$(Platform)` substitution in the .csproj.  Nevermind.

Reply:
I wasn't sure if using `$(Platform)` was too fancy or not. Thoughts?

Reply:
I think it is fine. We do something somewhat similar for ApiCompat baselines with TFMs, I think it is reasonable. That said here you are really only doing it for wasm so you could also just condition that so it is really just your choice.